src/components/DepositWidget/Row.tsx

```diff
@@ -75,6 +162,20 @@ export const Row: React.FC<RowProps> = (props: RowProps) => {
   const [visibleForm, showForm] = useState<'deposit' | 'withdraw' | void>()
   const exchangeBalanceTotal = exchangeBalance.add(depositingBalance)
 
+  // Checks innerWidth
+  let showResponsive = innerWidth < 500
```

Comment:
I understand `innerWidth` is coming from `windowSpecs` spread onto `Row` props, but I don't see it unpacked here.
Where's `innerWidth` defined?

Reply:
good catch, damn. just learned that `innerWidth` is in the global scope of Node... from `window.innerWidth` i guess. maybe i should change the name to avoid issues like this. tho it does work properly the way it was i changed it to explicit from props